prov/efa/test/efa_unit_test_ep.c

```diff
@@ -1041,7 +1037,6 @@ void test_efa_rdm_ep_user_zcpy_rx_unhappy_due_to_sas(struct efa_resource **state
 	assert_non_null(resource->hints);
 
 	resource->hints->tx_attr->msg_order = FI_ORDER_SAS;
-	resource->hints->rx_attr->msg_order = FI_ORDER_NONE;
```

Comment:
tx/rx msg order should be aligned. This was a miss when I added the test. Just make rx attr to be SAS as well here.